### ISSUE TITLE
[VT-48] add rc for report header effects (workaround for bug #1222)

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#70]
_Paul Beckingham on 2013-06-23T14:16:23Z says:_ 

 From the vit TODO file.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-06-23T14:16:23Z
Modified: 2015-03-01T02:43:05Z
```

**Comment by taskwarrior** [comment]
_David Patrick on 2014-06-08T20:24:23Z says:_ 

 what was bug #1222?
Does it still  need to be worked around?

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2015-03-01T02:43:05Z says:_ 

 Whatever bug #1222 was (under the old bug report system), it was fixed in version 2.3.0 (thanks to Paul for the pointer). Thus, a workaround does not seem necessary. If someone wants to know what #1222 was, git blame on the ChangeLog might have the answer.
